org.opent2t.onboarding.nest/js/README.md

```diff
@@ -0,0 +1,11 @@
+# Nest Onboarding
+Sample [Nest](http://www.nest.com/) onboarding module for Open Translators to Things. May be used to discover devices connected to Nest per the 
+[org.OpenT2T.Onboarding.Nest.xml](https://github.com/openT2T/onboarding/blob/master/org.OpenT2T.Onboarding.Nest/org.OpenT2T.Onboarding.Nest.xml) schema.
+
+See schema definition for inputs to the onboarding method, and outputs via success and error callbacks.
+
+## Setup
+1. Go to http://developer.nest.com and register as a developer
+2. Create an application (you will need the client_id and client_secret from this application)
```

Comment:
on developer.nest.com/products, the authorization URL still stays the same, but the web page that people will copy the ID and secret refers to them as "Product ID" and "Product Secret"

---
In reply to: [77262266](https://github.com/openT2T/onboarding/pull/13#discussion_r77262266) [](ancestors = 77262266)